doc/src/manual/environment-variables.md

```diff
@@ -0,0 +1,138 @@
+# Environment Variables in Julia
+
+The environment variables that Julia uses generally start with `JULIA`. If [`Base.versioninfo()`](https://github.com/JuliaLang/julia/blob/master/base/interactiveutil.jl#L247) is called with `verbose` equal to `true`, then the output will list all environment variables for which `JULIA` appears in the name.
```

Comment:
Typically we put line breaks to keep the line length down. Markdown renders a single newline as a space, so it works out just fine.